### PR TITLE
add ticdc summary monitor doc

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-summary-monitor.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- Transaction Sink：下游为 Mysql 或者 TiDB 时的写延迟信息
+- MQ Sink: 下游为 MQ 系统时的写延迟信息
+- Cloud Storage Sink：下游为 Cloud Storage 时的写速率信息
+- Redo：开启 Redo 功能时的写延迟信息
+
+## Server 监控栏 
+
+Server 监控栏示例如下：
+
+![TiCDC Summary Dashboard - Server metrics](/media/ticdc/ticdc-sammary-monitor-server.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:28]:
```suggestion
![TiCDC Summary Dashboard - Server metrics](/media/ticdc/ticdc-summary-monitor-server.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+- Uptime：TiCDC 节点已经运行的时间。
+- CPU usage：TiCDC 节点使用的 CPU。
+- Memory usage：TiCDC 节点使用的内存。
+
+## Changefeed 监控栏
+
+Changefeed 监控栏示例如下：
+
+![TiCDC Summary Dashboard - Changefeed metrics](/media/ticdc/ticdc-sammary-monitor-changefeed.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:38]:
```suggestion
![TiCDC Summary Dashboard - Changefeed metrics](/media/ticdc/ticdc-summary-monitor-changefeed.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+- Changefeed checkpoint lag：这个指标代表上游 TiDB 集群和下游系统之间的数据复制延迟，延迟是以时间为单位的。 通常情况下，该指标反映了 Changefeed 整体的数据同步状况是否健康，通常情况下，lag 越小，说明同步任务状态越好。而当 lag 上升时，通常说明 Changefeed 的同步能力或者下游系统的消费能力无法匹配上游的写入速度。
+- Changefeed resolved ts lag：这个指标代表了上游TiDB 集群与 TiCDC 节点之间的数据延迟，延迟是以时间为单位的。该指标能够反映 Changefeed 拉取上游数据变更的能力，当 lag 上升时，说明 Changefeed 无法及时的拉取上游产生的数据变更。
+
+## Dataflow 监控栏
+
+![TiCDC Summary Dashboard - Puller metrics](/media/ticdc/ticdc-sammary-monitor-dataflow-puller.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:45]:
```suggestion
![TiCDC Summary Dashboard - Puller metrics](/media/ticdc/ticdc-summary-monitor-dataflow-puller.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+- Puller output events/s：TiCDC 节点中 Puller 模块每秒输出到 Sorter 模块的数据变更个数。该指标能够反映 TiCDC 拉取上游变更事件的速度。
+- Puller output event：TiCDC 节点中 Puller 模块输出到 Sorter 模块的数据变更总数。
+
+![TiCDC Summary Dashboard - Sorter metrics](/media/ticdc/ticdc-sammary-monitor-dataflow-sorter.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:50]:
```suggestion
![TiCDC Summary Dashboard - Sorter metrics](/media/ticdc/ticdc-summary-monitor-dataflow-sorter.png)
```

Comment (shichun-0415) [reply ticdc/ticdc-summary-monitor.md:50]: Seems this figure is missing. Please upload it.

File: ticdc/ticdc-summary-monitor.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+- Sorter output events/s: TiCDC 节点中 Sorter 模块每秒输出到 Sink 模块的数据变更个数。值得注意的是，Sorter 的数据输出速率会受到 Sink 模块的影响，因此在发现 Sorter 模块输出速率比 Puller 模块低时，不一定是因为 Sorter 模块排序速度过慢；而应该先观察 Sink 模块的相关指标，确认是否是因为 Sink 模块 Flush 数据的耗时较长，导致 Sorter 模块输出降低。
+- Sorter output event: TiCDC 节点中 Sorter 模块输出到 Sorter 模块的数据变更总个数。
+
+![TiCDC Summary Dashboard - Mounter metrics](/media/ticdc/ticdc-sammary-monitor-dataflow-mounter.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:55]:
```suggestion
![TiCDC Summary Dashboard - Mounter metrics](/media/ticdc/ticdc-summary-monitor-dataflow-mounter.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+- Mounter output events/s：TiCDC 节点中 Mounter 模块每秒解码的数据变更的个数。当上游发生的数据变更包含较多字段时，Mounter 的解码速度可能会受到影响。
+- Mounter output event: TiCDC 节点中 Mounter 模块解码的数据变更的总个数。
+
+![TiCDC Summary Dashboard - Sink metrics](/media/ticdc/ticdc-sammary-monitor-dataflow-sink.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:60]:
```suggestion
![TiCDC Summary Dashboard - Sink metrics](/media/ticdc/ticdc-summary-monitor-dataflow-sink.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+- Sink flush rows/s: TiCDC 节点中 Sink 模块每秒往下游输出的数据变更个数。该指标反映的是同步任务向下游进行同步的速率，当 Sink flush rows/s 小于 Puller output events/s 时，同步延迟就可能会上升。
+- Sink flush rows: TiCDC 节点中 Sink 模块输出的数据变更的总个数。
+
+## Transaction Sink 监控栏
+
+Transaction Sink 监控栏示意如下，该监控栏只有下游为 MySQL 或者 TiDB 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-sammary-monitor-transaction-sink.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:69]:
```suggestion
![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-transaction-sink.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+- Backend Flush Duration: TiCDC Transaction Sink 模块在向下游执行一条 SQL 语句的耗时。通过观察该指标，能够判断下游的性能是否为同步速度的瓶颈。一般来说，p999 应该维持在 500 ms 内为佳，超过该值时，同步速度可能就会受到影响，引起 Checkpoint lag 上升。
+- Full Flush Duration: TiCDC 中每个事务从 Sorter 排序完成直到发送到下游之间的总耗时。用该值减去 Backend Flush Duration 的值，即可得出一个事务在被执行到下游之前的总排队时长。如果排队时长较高，可以考虑给同步任务分配更多的内存 Quota。
+
+## MQ Sink 监控栏
+
+MQ Sink 监控栏示意如下，该监控栏只有下游为 Kafka 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-sammary-monitor-mq-sink.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:78]:
```suggestion
![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-mq-sink.png)
```

Comment (shichun-0415) [reply ticdc/ticdc-summary-monitor.md:78]: Seems this figure is missing. Please upload it.

File: ticdc/ticdc-summary-monitor.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+- Worker Send Message Duration Percentile: TiCDC MQ Sink 的 Worker 往下游发送数据的延迟。
+- Kafka Ongoing Bytes: TiCDC MQ Sink 往下游发送数据的速率。
+
+## Cloud Storage Sink 监控栏
+
+Cloud Storage Sink 监控栏示意如下，该监控栏只有下游为 Cloud Storage 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-sammary-monitor-storage.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:87]:
```suggestion
![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-storage.png)
```

Comment (shichun-0415) [reply ticdc/ticdc-summary-monitor.md:87]: Seems this figure is missing. Please upload it.

File: ticdc/ticdc-summary-monitor.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+- Write Bytes/s: Cloud Storage Sink 模块往下游写数据的速率。
+- File Count: Cloud Storage Sink 模块写文件的总数量。
+
+## Redo 监控栏
+
+Redo 监控栏示意如下，该监控栏只有在开启了 Redo Log 功能时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-sammary-monitor-redo.png)
```
Comment (shichun-0415) [inline ticdc/ticdc-summary-monitor.md:96]:
```suggestion
![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-redo.png)
```

File: ticdc/ticdc-summary-monitor.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
+---
+title: TiCDC 基本监控指标
+summary: 了解 TiCDC 基本的监控指标。
+aliases: ['/zh/tidb/dev/ticdc-grafana-summary-dashboard']
```
[the remark this answers is not in the document]
Comment (asddongmen) [reply ticdc/ticdc-summary-monitor.md:4]: No, it is needed to keep both of them.

[the remark this answers is not in the document]
Comment (shichun-0415) [reply ticdc/ticdc-summary-monitor.md:4]: Then we should remove the aliases. An aliases is used only when a file is deleted.

[the remark this answers is not in the document]
Comment (asddongmen) [reply ticdc/ticdc-summary-monitor.md:4]: got it, thanks!

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-summary-monitor.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+---
+
+# TiCDC 基本监控指标
+
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:8]: 这里说是 7.0.0 的功能，应该不需要 cherry pick 到 6.6 和 6.5 的文档吧？

File: ticdc/ticdc-summary-monitor.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+- Redo：开启 Redo 功能时的写延迟信息
+
+## Server 监控栏 
+
+Server 监控栏示例如下：
+
+![TiCDC Summary Dashboard - Server metrics](/media/ticdc/ticdc-summary-monitor-server.png)
+
+- Uptime：TiCDC 节点已经运行的时间。
+- CPU usage：TiCDC 节点使用的 CPU。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:32]:
```suggestion
- CPU usage：TiCDC 节点的 CPU 使用量。
```

File: ticdc/ticdc-summary-monitor.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+- Memory usage：TiCDC 节点使用的内存。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:33]:
```suggestion
- Memory usage：TiCDC 节点的内存使用量。
```

File: ticdc/ticdc-summary-monitor.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+
+## Changefeed 监控栏
+
+Changefeed 监控栏示例如下：
+
+![TiCDC Summary Dashboard - Changefeed metrics](/media/ticdc/ticdc-summary-monitor-changefeed.png)
+
+- Changefeed checkpoint lag：这个指标代表上游 TiDB 集群和下游系统之间的数据复制延迟，延迟是以时间为单位的。 通常情况下，该指标反映了 Changefeed 整体的数据同步状况是否健康，通常情况下，lag 越小，说明同步任务状态越好。而当 lag 上升时，通常说明 Changefeed 的同步能力或者下游系统的消费能力无法匹配上游的写入速度。
+- Changefeed resolved ts lag：这个指标代表了上游TiDB 集群与 TiCDC 节点之间的数据延迟，延迟是以时间为单位的。该指标能够反映 Changefeed 拉取上游数据变更的能力，当 lag 上升时，说明 Changefeed 无法及时的拉取上游产生的数据变更。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:42]:
```suggestion
- Changefeed checkpoint lag：这个指标代表上游 TiDB 集群和下游系统之间的数据复制延迟，延迟以时间为单位。 通常情况下，该指标反映了 Changefeed 整体的数据同步状况是否健康，通常情况下，lag 越小，说明同步任务状态越好。而当 lag 上升时，通常说明 Changefeed 的同步能力或者下游系统的消费能力无法匹配上游的写入速度。
- Changefeed resolved ts lag：这个指标代表了上游TiDB 集群与 TiCDC 节点之间的数据延迟，延迟以时间为单位。该指标能够反映 Changefeed 拉取上游数据变更的能力，当 lag 上升时，说明 Changefeed 无法及时地拉取上游产生的数据变更。
```

File: ticdc/ticdc-summary-monitor.md [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+## Dataflow 监控栏
+
+![TiCDC Summary Dashboard - Puller metrics](/media/ticdc/ticdc-summary-monitor-dataflow-puller.png)
+
+- Puller output events/s：TiCDC 节点中 Puller 模块每秒输出到 Sorter 模块的数据变更个数。该指标能够反映 TiCDC 拉取上游变更事件的速度。
+- Puller output events：TiCDC 节点中 Puller 模块输出到 Sorter 模块的数据变更总数。
+
+![TiCDC Summary Dashboard - Sorter metrics](/media/ticdc/ticdc-summary-monitor-dataflow-sorter.png)
+
+- Sorter output events/s: TiCDC 节点中 Sorter 模块每秒输出到 Sink 模块的数据变更个数。值得注意的是，Sorter 的数据输出速率会受到 Sink 模块的影响，因此在发现 Sorter 模块输出速率比 Puller 模块低时，不一定是因为 Sorter 模块排序速度过慢；而应该先观察 Sink 模块的相关指标，确认是否是因为 Sink 模块 Flush 数据的耗时较长，导致 Sorter 模块输出降低。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:53]:
```suggestion
- Sorter output events/s：TiCDC 节点中 Sorter 模块每秒输出到 Sink 模块的数据变更个数。值得注意的是，Sorter 的数据输出速率会受到 Sink 模块的影响，因此在发现 Sorter 模块输出速率比 Puller 模块低时，不一定是因为 Sorter 模块排序速度过慢；而应该先观察 Sink 模块的相关指标，确认是否是因为 Sink 模块 Flush 数据的耗时较长，导致 Sorter 模块输出降低。
```

File: ticdc/ticdc-summary-monitor.md [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+- Sorter output event: TiCDC 节点中 Sorter 模块输出到 Sorter 模块的数据变更总个数。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:54]:
```suggestion
- Sorter output event：TiCDC 节点中 Sorter 模块输出到 Sorter 模块的数据变更总个数。
```

File: ticdc/ticdc-summary-monitor.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+
+![TiCDC Summary Dashboard - Mounter metrics](/media/ticdc/ticdc-summary-monitor-dataflow-mounter.png)
+
+- Mounter output events/s：TiCDC 节点中 Mounter 模块每秒解码的数据变更的个数。当上游发生的数据变更包含较多字段时，Mounter 的解码速度可能会受到影响。
+- Mounter output event: TiCDC 节点中 Mounter 模块解码的数据变更的总个数。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:59]:
```suggestion
- Mounter output event：TiCDC 节点中 Mounter 模块解码的数据变更的总个数。
```

File: ticdc/ticdc-summary-monitor.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+
+![TiCDC Summary Dashboard - Sink metrics](/media/ticdc/ticdc-summary-monitor-dataflow-sink.png)
+
+- Sink flush rows/s: TiCDC 节点中 Sink 模块每秒往下游输出的数据变更个数。该指标反映的是同步任务向下游进行同步的速率，当 Sink flush rows/s 小于 Puller output events/s 时，同步延迟就可能会上升。
+- Sink flush rows: TiCDC 节点中 Sink 模块输出的数据变更的总个数。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:64]:
```suggestion
- Sink flush rows/s：TiCDC 节点中 Sink 模块每秒往下游输出的数据变更个数。该指标反映的是同步任务向下游进行同步的速率，当 Sink flush rows/s 小于 Puller output events/s 时，同步延迟可能会上升。
- Sink flush rows：TiCDC 节点中 Sink 模块输出的数据变更的总个数。
```

File: ticdc/ticdc-summary-monitor.md [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+
+## Transaction Sink 监控栏
+
+Transaction Sink 监控栏示意如下，该监控栏只有下游为 MySQL 或者 TiDB 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-transaction-sink.png)
+
+- Backend Flush Duration: TiCDC Transaction Sink 模块在向下游执行一条 SQL 语句的耗时。通过观察该指标，能够判断下游的性能是否为同步速度的瓶颈。一般来说，p999 应该维持在 500 ms 内为佳，超过该值时，同步速度可能就会受到影响，引起 Checkpoint lag 上升。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:72]:
```suggestion
- Backend Flush Duration：TiCDC Transaction Sink 模块在向下游执行一条 SQL 语句的耗时。通过观察该指标，能够判断下游的性能是否为同步速度的瓶颈。一般来说，p999 应该维持在 500 ms 内为佳，超过该值时，同步速度可能就会受到影响，引起 Changefeed checkpoint lag 上升。
```

File: ticdc/ticdc-summary-monitor.md [hunk 21]
```diff
@@ -0,0 +1,102 @@
...
+- Full Flush Duration: TiCDC 中每个事务从 Sorter 排序完成直到发送到下游之间的总耗时。用该值减去 Backend Flush Duration 的值，即可得出一个事务在被执行到下游之前的总排队时长。如果排队时长较高，可以考虑给同步任务分配更多的内存 Quota。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:73]:
```suggestion
- Full Flush Duration：TiCDC 中每个事务从 Sorter 排序完成直到发送到下游之间的总耗时。用该值减去 Backend Flush Duration 的值，即可得出一个事务在被执行到下游之前的总排队时长。如果排队时长较高，可以考虑给同步任务分配更多的内存 Quota。
```

File: ticdc/ticdc-summary-monitor.md [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+
+## MQ Sink 监控栏
+
+MQ Sink 监控栏示意如下，该监控栏只有下游为 Kafka 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-mq-sink.png)
+
+- Worker Send Message Duration Percentile: TiCDC MQ Sink 的 Worker 往下游发送数据的延迟。
+- Kafka Ongoing Bytes: TiCDC MQ Sink 往下游发送数据的速率。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:82]:
```suggestion
- Worker Send Message Duration Percentile：TiCDC MQ Sink 的 Worker 往下游发送数据的延迟。
- Kafka Ongoing Bytes：TiCDC MQ Sink 往下游发送数据的速率。
```

File: ticdc/ticdc-summary-monitor.md [hunk 23]
```diff
@@ -0,0 +1,102 @@
...
+
+## Cloud Storage Sink 监控栏
+
+Cloud Storage Sink 监控栏示意如下，该监控栏只有下游为 Cloud Storage 时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-cloud-storage.png)
+
+- Write Bytes/s: Cloud Storage Sink 模块往下游写数据的速率。
+- File Count: Cloud Storage Sink 模块写文件的总数量。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:91]:
```suggestion
- Write Bytes/s：Cloud Storage Sink 模块往下游写数据的速率。
- File Count：Cloud Storage Sink 模块写文件的总数量。
```

File: ticdc/ticdc-summary-monitor.md [hunk 24]
```diff
@@ -0,0 +1,102 @@
...
+
+## Redo 监控栏
+
+Redo 监控栏示意如下，该监控栏只有在开启了 Redo Log 功能时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-redo.png)
+
+- Redo Write rows/s: Redo 模块每秒写数据的行数。当开启 Redo 功能时，如果发现同步任务的延迟上升，那么可以观察该指标是否和 Puller Output event/s 的值相差较大。
```
Comment (ran-huang) [inline ticdc/ticdc-summary-monitor.md:99]: 如果相差较大的话，说明...？

File: ticdc/ticdc-summary-monitor.md (hunk 12 above)
Comment (asddongmen) [reply ticdc/ticdc-summary-monitor.md:7]: 不用

File: ticdc/ticdc-summary-monitor.md
[the remark this answers is not in the document]
Comment (asddongmen) [reply ticdc/ticdc-summary-monitor.md:1]: done PTAL~

**Review by ran-huang — APPROVED** [review]

File: ticdc/ticdc-summary-monitor.md [hunk 25]
```diff
@@ -0,0 +1,101 @@
...
+## Redo 监控栏
+
+Redo 监控栏示意如下，该监控栏只有在开启了 Redo Log 功能时才有数据。
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-redo.png)
+
+- Redo Write rows/s: Redo 模块每秒写数据的行数。当开启 Redo 功能时，如果发现同步任务的延迟上升，那么可以观察该指标是否和 Puller Output event/s 的值有较大差距。如果是，那么可能是由于 Redo 模块的写入能力不足造成延迟上升。
+- Redo Write Byte/s: Redo 模块每秒写数据的速率。
+- Redo flush log duration: Redo 模块往下游刷写数据的耗时。若该指标值较高，则可能是它影响了同步的速度。
+- Redo flushall duration: 数据变更停留在 Redo 模块中的总时长。
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:101]:
```suggestion
- Redo Write rows/s：Redo 模块每秒写数据的行数。当开启 Redo 功能时，如果发现同步任务的延迟上升，那么可以观察该指标是否和 Puller Output event/s 的值有较大差距。如果是，那么可能是由于 Redo 模块的写入能力不足造成延迟上升。
- Redo Write Byte/s：Redo 模块每秒写数据的速率。
- Redo flush log duration：Redo 模块往下游刷写数据的耗时。若该指标值较高，则可能是它影响了同步的速度。
- Redo flushall duration：数据变更停留在 Redo 模块中的总时长。
```

File: TOC.md [hunk 26]
```diff
@@ -534,7 +534,8 @@
       - [日志过滤器](/ticdc/ticdc-filter.md)
       - [双向复制](/ticdc/ticdc-bidirectional-replication.md)
     - 监控告警
-      - [监控指标](/ticdc/monitor-ticdc.md)
+      - [基本监控指标](/ticdc/ticdc-summary-monitor.md)
+      - [详细监控指标](/ticdc/monitor-ticdc.md)
```
Comment (qiancai) [inline TOC.md:538]: /ticdc/monitor-ticdc.md 里目前的 title 是“TiCDC 重要监控指标详解"，是否也需要改为 “TiCDC 详细监控指标”

**Review by qiancai — APPROVED** [review]
other LGTM

File: TOC.md (hunk 26 above)
Comment (benmaoer) [reply TOC.md:538]: 好建议

Comment (asddongmen) [reply TOC.md:538]: thank you!  I will change it later.

Comment (asddongmen) [reply TOC.md:538]: done @qiancai  PTAL

**Review by benmaoer — APPROVED** [review]
LGTM

**Pull request merged**